product-features/liquidity-pools/faqs.md

```diff
@@ -99,3 +101,91 @@ However, if Carol had simply held onto her 100 STX and 150 sUSDT, their combined
+
+</details>
+
+## Self-Service Listing
+
+<details>
+
+<summary>Which are the anchor tokens supported?</summary>
+
+Native STX currency, ALEX token and aBTC token.
+
+</details>
+
+<details>
+
+<summary>Is there a minimum liquidity amount for the anchor token?</summary>
+
+Yes. Initial liquidity of the anchor token must be a minimum of 1,800 STX or the equivalent value in in ALEX or aBTC tokens.
+
+</details>
+
+<details>
+
+<summary>How is the listed token initial price determined?</summary>
+
+The initial price is determined by the initial liquidity provided by the creator. The ratio between the pair of funds determines the price relationship between both tokens. 
+
+For instance, if the creator provides 8,000 listing tokens and 2,000 anchor tokens, that means the initial ratio is 4:1. Note that pool ratios are calculated as the minimal expression of the fraction between the token balances. In this case, is 8,000 / 2,000.
+
+We can think of this initial ratio in two ways (and they are both equivalent):
+
+- 4 listing tokens equals 1 anchor token.
+- 1 listing token equals 0.25 anchor tokens.
+
+</details>
+
+<details>
+
+<summary>How does the price of the listing token change once the pool is created?</summary>
+
+Once the pool is created, the price discovery phase begins. Users can permissionlessly trade the pair of assets, and the [Automated Market Maker (AMM)](../../detailed-information/alexs-automated-market-maker-amm.md) algorithm will determine the price dynamics of the newly listed token. For further information on this topic please refer to the [ALEXGo Trading Pool documentation](https://docs.alexgo.io/automated-market-making/trading-pool).
+
+</details>
+
+<details>
+
+<summary>Is there some requirement to make the listed token visible in ALEX Token List?</summary>
+
+Yes. ALEX requires a [Coingecko](https://www.coingecko.com/) or [CoinMarketCap](https://coinmarketcap.com/) token listing to verify the provided social media information before uploading it to the official list at [app.alexlab.co/token-list](https://app.alexlab.co/token-list).
+
+Once that is done, click on `Customer Support` on the [Self-Service Listing page](https://app.alexlab.co/self-service-listing) or contact us via Telegram at [t.me/ALEXselfservice ](https://t.me/ALEXselfservice) to submit your information (e.g. X accont, Discord, official website).
+
+</details>
+
+<details>
+
+<summary>What is a wrapped version of a token contract?</summary>
+
+Wrapped token contracts refer to "pass-through" tokens that don't retain economics; their purpose is to simplify development and enhance security. ALEX is responsible for deploying wrapped contracts. As its primarily technical, it is not relevant from a user perspective rather than it involves a whole step in the procedure and takes some time.
```

Comment:
Wrapped token contracts refer to "pass-through" tokens that don't retain economics; their purpose is to simplify development and enhance security. ALEX is responsible for deploying wrapped contracts. As its primarily technical, it is not relevant from a user perspective other than it involves a whole step in the procedure and takes some time.